models/order/src/parcel/generator.ts

```diff
@@ -0,0 +1,24 @@
+import type { Parcel } from '@commercetools/platform-sdk';
+import { fake, Generator } from '@commercetools-test-data/core';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import { DeliveryItem } from '../delivery-item';
+import { ParcelMeasurements } from '../parcel-measurements';
+import { TrackingData } from '../tracking-data';
+
+const [getOlderDate] = createRelatedDates();
+
+// https://docs.commercetools.com/api/projects/orders#Parcel
```

Comment:
```suggestion
// https://docs.commercetools.com/api/projects/orders#parcel
```
The capitalized `P` doesn't navigate to the parcel section